### PR TITLE
[MISC] Artificer Infusion Update + Small Fixes

### PR DESCRIPTION
**Pull request by Azzzzzy**
Split the Artificer Infusions into their own supplements
Added Missing TCOE Infusions
Slight Renaming to Spellwrought Tattoo to fit new naming Scheme
Fixed Summon Fiend Spirit Statblock ID
Fixed Tiefling ASI with variant subraces

### CODE REVIEW

File: supplements/waterdeep-dungeon-of-the-mad-mage.index [hunk 1]
```diff
@@ -9,5 +9,6 @@
     <files>
         <file name="source.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/waterdeep-dungeon-of-the-mad-mage/source.xml" />
         <file name="items.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/waterdeep-dungeon-of-the-mad-mage/items.xml" />
+        <file name="replicate-infusions.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/waterdeep-dungeon-of-the-mad-mage/replicate-infusions.xml" />
```
Comment (kgirtxd) [inline supplements/waterdeep-dungeon-of-the-mad-mage.index:12]: Forgot to increment version number

File: supplements/tashas-cauldron-of-everything/replicate-infusions.xml [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+	<element name="Armor of Magical Strength" type="Class Feature" source="Tasha’s Cauldron of Everything" id="ID_WOTC_ERLW_CLASS_FEATURE_ARTIFICER_INFUSION_ARMOR_OF_MAGICAL_STRENGTH">
+		<supports>Artificer Infusion</supports>
+		<description>
+			<p><em>Item: A suit of armor (requires attunement)</em></p>
+			<p>This armor has 6 charges. The wearer can expend the armor's charges in the following ways: </p>
+			<ul>
+			<li>When the wearer makes a Strength check or a Strength saving throw, it can expend 1 charge to add a bonus to the roll equal to its Intelligence modifier.</li>
+			<li>If the creature would be knocked prone, it can use its reaction to expend 1 charge to avoid being knocked prone.</li>
+			</ul>
+			<p>The armor regains ld6 expended charges daily at dawn.</p>
```
Comment (kgirtxd) [inline supplements/tashas-cauldron-of-everything/replicate-infusions.xml:115]: 1d6 not ld6

File: supplements/tashas-cauldron-of-everything/replicate-infusions.xml [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+	</element>
+
+	<element name="Armor of Magical Strength" type="Magic Item" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_MAGIC_ITEM_ARMOR_ARMOR_OF_MAGICAL_STRENGTH">
+		<description>
+			<p>This armor has 6 charges. The wearer can expend the armor's charges in the following ways: </p>
+			<ul>
+			<li>When the wearer makes a Strength check or a Strength saving throw, it can expend 1 charge to add a bonus to the roll equal to its Intelligence modifier.</li>
+			<li>If the creature would be knocked prone, it can use its reaction to expend 1 charge to avoid being knocked prone.</li>
+			</ul>
+			<p>The armor regains ld6 expended charges daily at dawn.</p>
```
Comment (kgirtxd) [inline supplements/tashas-cauldron-of-everything/replicate-infusions.xml:132]: 1d6 again

File: supplements/tashas-cauldron-of-everything/replicate-infusions.xml [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+			</div>
+		</description>
+		<sheet>
+			<description>You can infuse a suit of armor or robes into a Mind Sharpener.</description>
+		</sheet>
+	</element>
+	
+	<element name="Mind Sharpener" type="Magic Item" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_MAGIC_ITEM_ARMOR_MIND_SHARPENER">
+		<description>
+			<p>The infused item can send a jolt to the wearer to refocus their mind. The item has 4 charges. When the wearer fails a Constitution saving throw to maintain concentration on a spell, the wearer can use its reaction to expend 1 of the item's charges to succeed instead. The item regains ld4 expended charges daily at dawn.</p>
```
Comment (kgirtxd) [inline supplements/tashas-cauldron-of-everything/replicate-infusions.xml:193]: 1d4

File: supplements/tashas-cauldron-of-everything/replicate-infusions.xml [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+			<set name="cost" currency="gp">0</set>
+			<set name="charges">4</set>
+			<set name="type" addition="light, medium, or heavy">Armor</set>
+			<set name="armor">ID_INTERNAL_ARMOR_GROUP_LIGHT|ID_INTERNAL_ARMOR_GROUP_MEDIUM|ID_INTERNAL_ARMOR_GROUP_HEAVY</set>
+			<set name="name-format">Mind Sharpener {{parent}}</set>
+		</setters>
+	</element>
+	<element name="Mind Sharpener Robe" type="Magic Item" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_MAGIC_ITEM_MIND_SHARPENER">
+		<description>
+			<p>The infused item can send a jolt to the wearer to refocus their mind. The item has 4 charges. When the wearer fails a Constitution saving throw to maintain concentration on a spell, the wearer can use its reaction to expend 1 of the item's charges to succeed instead. The item regains ld4 expended charges daily at dawn.</p>
```
Comment (kgirtxd) [inline supplements/tashas-cauldron-of-everything/replicate-infusions.xml:207]: 1d4 again

File: supplements/tashas-cauldron-of-everything/replicate-infusions.xml [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+		<supports>Artificer Infusion</supports>
+		<requirements>[level:14]</requirements>
+		<prerequisite>14th-level artificer</prerequisite>
+		<description>
+			<p><em>Prerequisite: 14th-level artificer</em></p>
+			<p><em>Item: A suit of armor (requires attunement)</em></p>
+			<p>The wearer of this armor gains these benefits: </p>
+			<ul>
+			<li>The wearer's walking speed increases by 5 feet.</li>
+			<li>The armor includes gauntlets, each of which is a magic melee weapon that can be wielded only when the hand is holding nothing. The wearer is proficient with the gauntlets, and each one deals ld8 force damage on a hit and has the thrown property, with a normal range of 20 feet and a long range of 60 feet. When thrown, the gauntlet detaches and flies at the attack's target, then immediately returns to the wearer and reattaches.</li>
```
Comment (kgirtxd) [inline supplements/tashas-cauldron-of-everything/replicate-infusions.xml:71]: 1d8 not ld8

File: supplements/tashas-cauldron-of-everything/replicate-infusions.xml [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+			<description>You can infuse a suit of armor into Arcane Propulsion Armor.</description>
+		</sheet>
+	</element>
+
+	<element name="Arcane Propulsion Armor" type="Magic Item" source="Tasha’s Cauldron of Everything" id="ID_WOTC_TCOE_MAGIC_ITEM_ARMOR_ARCANE_PROPULSION_ARMOR">
+		<description>
+			<p>The wearer of this armor gains these benefits: </p>
+			<ul>
+			<li>The wearer's walking speed increases by 5 feet.</li>
+			<li>The armor includes gauntlets, each of which is a magic melee weapon that can be wielded only when the hand is holding nothing. The wearer is proficient with the gauntlets, and each one deals ld8 force damage on a hit and has the thrown property, with a normal range of 20 feet and a long range of 60 feet. When thrown, the gauntlet detaches and flies at the attack's target, then immediately returns to the wearer and reattaches.</li>
```
Comment (kgirtxd) [inline supplements/tashas-cauldron-of-everything/replicate-infusions.xml:89]: 1d8 again

**Review by kgirtxd — CHANGES_REQUESTED** [review]

**Review by kgirtxd — CHANGES_REQUESTED** [review]
Change IDs in the TCoE infusions from ERLW to TCOE.  They are not currently in the builder so it should not break anything.  I think we should leave the rest of the IDs as they are.

**Review by kgirtxd — APPROVED** [review]

**Review by AngryGuineaPig — APPROVED** [review]

**Review by NickVendel — CHANGES_REQUESTED** [review]

**Review by NickVendel — APPROVED** [review]

**Pull request merged**